tests/test_templates.py

```diff
@@ -0,0 +1,71 @@
+import logging
+import os
+import unittest
+
+import numpy as np
+
+from pint.templates import lcprimitives
+from pint.templates import lctemplate
+
+
+def gauss(x, x0, s):
+    return 1.0 / s / (2 * np.pi) ** 0.5 * np.exp(-0.5 * (x - x0) ** 2 / s ** 2)
+
+
+class TestPrimitives(unittest.TestCase):
+    """ Test instantiation and various properties of LCPrimitive objects.
+    """
+
+    @classmethod
+    def setUpClass(cls):
+        pass
+
+    def test_basic_functionality(self):
+        """ Make sure objects adequately implement mathematical intent."""
+
+        # instantiate a basic gaussian with known parameters and make sure
+        # it "looks" like a gaussian.  The main thing here is that these
+        # functions are wrapped.
+
+        g1 = lcprimitives.LCGaussian(p=[0.01, 0.5])
+        # defined as 1/sigma/(2*pi)^0.5 * (-0.5*(x-x0)^2) WRAPPED
+        # because this is a narrow gaussian, it's essentially the same as
+        # an unwrapped version
+        expected_val = gauss(0.50, 0.5, 0.01)
+        assert abs(g1(0.50) - expected_val) < 1e-6
+        expected_val = gauss(0.48, 0.5, 0.01)
+        assert abs(g1(0.48) - expected_val) < 1e-6
+
+        # explicitly test wrapping
```

Comment:
It's nicer to create a separate fest function for testing separate properties like this, so bugs can be more easily localized.